.ci/deps.sh

```diff
@@ -67,8 +67,8 @@ fi
 
 deps_perl="perl libperl-critic-perl"
 
-sudo apt-get -y update
-sudo apt-get -y --no-install-recommends install $deps $deps_perl $deps_infer
+sudo apt-get -y update|grep -v '\%'
```

Comment:
@yukiisbored Setting TERM environment to dumb will have effect on console window but will it have any effect on log file because @Makman2 had pointed out this.
![image](https://cloud.githubusercontent.com/assets/17270966/22437372/d6ee77b4-e74d-11e6-9dc5-d41c8f55fa0c.png)